src/main/webapp/stylesheets/teammatesCommon.css

```diff
@@ -1173,7 +1173,8 @@ Glyphicon for success status messages
 .alert-success:before {
     font-family: 'Glyphicons Halflings';
     display: inline-block;
-    padding-right: 0.5em;
+    padding-top: .2em;
```

Comment:
> I'm not sure how many modals we'd need to remove the alert class from though; could either of you do a quick scan

- I believe a lot of the smaller modals can be taken care of by removing the alert classes from the bootbox implementations.
- Other than that, not sure how to do that kind of search across the codebase 😅